### PR TITLE
checking

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: applications/e-commerce/Sample.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            </ul>
+        </main>
+
+        <footer>
+            Contact, Map/Location, other links...
+        </footer>
+        
+
+    </body>
+</head>
```
Comment (tupes) [inline applications/e-commerce/Sample.html]: Your content shouldn't go in the `<head>` element.

File: applications/e-commerce/Sample.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<head>
```
Comment (tupes) [inline applications/e-commerce/Sample.html:2]: Missing `DOCTYPE` declaration and `<html>` element.

File: applications/e-commerce/Sample.html (hunk 1 above)
Comment (ghost) [reply applications/e-commerce/Sample.html]: oohhh... I think I should go through before submit it.... :D 
Thank you Mark...

File: applications/e-commerce/Ankesh.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<h1>
+    Hello
+    Just checking pull request works or not ? :D
+</h1>
```
Comment (tupes) [inline applications/e-commerce/Ankesh.html]: Please remove this file from your PR. It's not part of the e-commerce application, and you already added a file in the `members` folder 😄

**Review by tupes — CHANGES_REQUESTED** [review]
The `Sample.html` file isn't exactly what I'm looking for, since it's not part of the e-commerce application that we're building, but I'd still approve this if you made these changes 😄

File: applications/e-commerce/Ankesh.html (hunk 3 above)
Comment (ghost) [reply applications/e-commerce/Ankesh.html]: Can you tell me how to remove it from PR ???

**Review by sprihozhiy — COMMENTED** [review]
Just a little recommendation. If you use semantic HTML5 elements (header, footer, main, aside, etc.) it is better to adhere to this practice for all of your code. You've wrapped the "Clothing" section with `<aside>,`  but your "Electronics" section is not wrapped with any semantic element. Technically, it won't be a problem. But this is not good for the website's SEO

**Review by tupes — APPROVED** [review]
I'm not sure how to incorporate this into the Sports Store, but good job on it! 😄

**Pull request merged**